packages/headless/src/features/search-parameters/search-parameter-serializer.ts

```diff
@@ -246,7 +247,10 @@ function processObjectValues(key: string, values: string[]) {
   return values;
 }
 
-export function buildNumericRanges(ranges: string[]) {
+export function buildNumericRanges(
+  ranges: string[],
+  state: FacetValueState = 'selected'
```

Comment:
Note to self: eventually we should remove this default value and always pass the correct state when calling this function.

Create a Jira to deal with this.

Reply:
it was a quick win so I did it here instead